pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
+name = "black-magic-org-git"
+version = "0.1.0"
+requires-python = ">=3.14"
```

Comment:
Is this strictly required? Because here we're on Python 3.13 and this would prevent us being able to build the website locally when we can right now.

Reply:
Agreed with Aki - 3.11 was working to build this just prior to the Debian upgrade we did, so should be a decent default for minimum Python version.

Reply:
ok, sounds good, will make the adjustment shortly

Reply:
ok, that is now done too :)